components/server/ee/src/billing/billing-service.ts

```diff
@@ -43,8 +44,8 @@ export class BillingService {
         }
 
         const upcomingInvoice = await this.getUpcomingInvoice(attributionId);
-        const currentInvoiceCredits = upcomingInvoice.getCredits();
-        if (currentInvoiceCredits >= costCenter.spendingLimit) {
+        const currentInvoiceCredits = upcomingInvoice.credits | 0;
+        if (currentInvoiceCredits >= (costCenter.spendingLimit || 0)) {
```

Comment:
rather than `|| 0` should we use the same `defaultSpendingLimit` as when the Stripe subscription is configured?

See [this comment](https://github.com/gitpod-io/gitpod/pull/12887#discussion_r968555099).